device/inventec/x86_64-inventec_d6356-r0/INVENTEC-D6356/port_config.ini

```diff
@@ -0,0 +1,57 @@
+# name          lanes             alias	        index
+Ethernet0       1                 Ethernet0     0
```

Comment:
index must start from 1.

Reply:
according to https://github.com/Azure/SONiC/wiki/Porting-Guide
we can set the index from 0 if our front panel ports are starting from 0

Reply:
in sfputil.py, this platform port mapping is from 0